FrontDesk/src/FrontDesk.Api/appsettings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "baseUrls": {
+    "apiBase": "https://localhost:44388/api/",
+    "webBase": "https://localhost:44394/"
+  },
+
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=FrontDesk.Web-20130819200722;",
+    "VetOfficeContext": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=FrontDesk.Web-20130819200722;",
+    "ServiceBroker": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=ServiceBrokerTest;"
```

Comment:
Same as above. Just need DefaultConnection for now.

Reply:
Done